packages/calypso-e2e/src/data-helper.ts

```diff
@@ -363,3 +333,39 @@ export function createSuiteTitle( title: string ): string {
 
 	return parts.join( ' ' );
 }
+
+/**
+ * Returns the Magnificent 16 test locales, known as Mag16.
+ *
+ * @returns {string[]} Array of strings in ISO-639-1 standard.
+ */
+export function getMag16Locales(): string[] {
+	return [
+		'en',
+		'es',
+		'pt-br',
+		'de',
+		'fr',
+		'he',
+		'ja',
+		'it',
+		'nl',
+		'ru',
+		'tr',
+		'id',
+		'zh-cn',
+		'zh-tw',
+		'ko',
+		'ar',
+		'sv',
+	];
+}
+
+/**
+ * Returns the list of viewports that are available for the framework.
+ *
+ * @returns {string([])} Array of strings for valid viewports.
+ */
+export function getViewports(): string[] {
+	return [ 'mobile', 'desktop' ];
+}
```

Comment:
I decided to move the "knowledge" of the Mag16 Locales and Viewports out of EnvVars to DataHelper, but as you can see later in the PR I did not follow this convention when dealing with SecretsManager.

To be honest, I am a little uneasy with this as well. I've looked at a few other OSS that use TypeScript (eg. [Jest](https://github.com/facebook/jest/blob/bb0956589f9dbddcc13db0f42b2d4d9ff642b2cd/packages/jest-expect/src/types.ts)) and I don't feel easy about defining and exporting a `const` from the types file. 

At the same time, the way our framework uses the values of a defined array in-code and converted into type means we can't get away from having a `const` defined  _somewhere_.